File: JWTExample/Entities/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JWTExample.Entities;

public class Student
{
    public int Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Email { get; set; }
}

File: JWTExample/ClientApp/src/app/Student.ts
export class Student {
  id: number;
  firstName: string;
  lastName: string;
  email: string;

  constructor(
    id: number = 0,
    firstName: string = '',
    lastName: string = '',
    email: string = ''
  ) {
    this.id = id;
    this.firstName = firstName;
    this.lastName = lastName;
    this.email = email;
  }
}
